hail/python/hailtop/aiocloud/aioazure/client/graph_client.py

```diff
@@ -5,6 +5,12 @@
 
 
 class AzureGraphClient(AzureBaseClient):
+    required_scopes = ['https://graph.microsoft.com/.default']
+
     def __init__(self, session: Optional[AzureSession] = None, **kwargs):
+        if 'scopes' in kwargs:
+            kwargs['scopes'] += AzureGraphClient.required_scopes
+        else:
+            kwargs['scopes'] = AzureGraphClient.required_scopes
```

Comment:
I made this change because I was prototyping locally and `AzureGraphClient` is unusable without the graph scope.